staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go

```diff
@@ -260,6 +263,14 @@ type DeploymentControllerConfiguration struct {
 	DeploymentControllerSyncPeriod metav1.Duration
 }
 
+// StatefulSetControllerConfiguration contains elements describing StatefulSetController.
+type StatefulSetControllerConfiguration struct {
+	// concurrentStatefulSetSyncs is the number of statefulset objects that are
+	// allowed to sync concurrently. Larger number = more responsive statefulsets,
+	// but more CPU (and network) load.
+	ConcurrentStatefulSetSyncs int32
```

Comment:
Name feels a little redundant, but follows the ConcurrentNamespaceSyncs / ConcurrentServiceSyncs pattern so :+1:

Reply:
@justinsb Yeah, I just follow the pattern `ConcurrentXXXXXSyncs` :smile: